Kudu.Core/Jobs/JobLogger.cs

```diff
@@ -68,6 +68,20 @@ protected string GetStatusFilePath()
         public void ReportStatus<TJobStatus>(TJobStatus status) where TJobStatus : class, IJobStatus
         {
             ReportStatus(status, logStatus: true);
+            if (status.GetType() == typeof(ContinuousJobStatus))
```

Comment:
I'm not seeing the change. Did you push?

The conditionals don't feel very polymorphic though, compare to calling virtual method which does the right thing in each derived class.